### PR TITLE
🌐 [i18n-KO] Translated `main_classes/onnx.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `main_classes/onnx.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/main_classes/onnx.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+[[autodoc]] onnx.config.OnnxConfigWithPast
+
+### OnnxSeq2SeqConfigWithPast[[OnnxSeq2SeqConfigWithPast]]
+
+[[autodoc]] onnx.config.OnnxSeq2SeqConfigWithPast
+
+## ONNX 특징[[onnx-features]]
+
+각 ONNX 설정은 다양한 유형의 토폴로지나 작업에 대해 모델을 내보낼 수 있게 해주는 _features_ 세트와 연관되어 있습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:45]:
```suggestion
각 ONNX 설정은 다양한 유형의 토폴로지나 작업의 모델을 익스포트 해주는 _features_ 세트와 관련이 있습니다.
```
export가 glossary에 안올라와 있어서, 임의로 익스포트로 바꿔보았습니다!

File: docs/source/ko/main_classes/onnx.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 패키지를 통해 설정 객체를 활용함으로써 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
+
+🤗 트랜스포머에 대한 자세한 내용은 [이 가이드](../serialization)를 참조하세요.
+
+## ONNX 설정[[onnx-configurations]]
+
+내보내고자 하는 모델 아키텍처의 유형에 따라 상속받아야 할 세 가지 추상 클래스를 제공합니다:
+
+* 인코더 기반 모델은 [`~onnx.config.OnnxConfig`]에서 상속받습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:27]:
```suggestion
* 인코더 기반 모델은 [`~onnx.config.OnnxConfig`]을 상속받습니다.
```

File: docs/source/ko/main_classes/onnx.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* 디코더 기반 모델은 [`~onnx.config.OnnxConfigWithPast`]에서 상속받습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:28]:
```suggestion
* 디코더 기반 모델은 [`~onnx.config.OnnxConfigWithPast`]을 상속받습니다.
```

File: docs/source/ko/main_classes/onnx.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+* 인코더-디코더 기반 모델은 [`~onnx.config.OnnxSeq2SeqConfigWithPast`]에서 상속받습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:29]:
```suggestion
* 인코더-디코더 기반 모델은 [`~onnx.config.OnnxSeq2SeqConfigWithPast`]을 상속받습니다.
```

File: docs/source/ko/main_classes/onnx.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+# 🤗 트랜스포머 모델을 ONNX로 내보내기[[exporting--transformers-models-to-onnx]]
+
+🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 패키지를 통해 설정 객체를 활용함으로써 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
+
+🤗 트랜스포머에 대한 자세한 내용은 [이 가이드](../serialization)를 참조하세요.
+
+## ONNX 설정[[onnx-configurations]]
+
+내보내고자 하는 모델 아키텍처의 유형에 따라 상속받아야 할 세 가지 추상 클래스를 제공합니다:
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:25]:
```suggestion
익스포트하고자하는 모델 아키텍처 유형에 따라 상속받아야 할 세 가지 추상 클래스를 제공합니다:
```

File: docs/source/ko/main_classes/onnx.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 🤗 트랜스포머 모델을 ONNX로 내보내기[[exporting--transformers-models-to-onnx]]
+
+🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 패키지를 통해 설정 객체를 활용함으로써 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/onnx.md:19]:
```suggestion
🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 이 패키지는 설정 객체를 활용하여 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
```

**Review by ahnjj — CHANGES_REQUESTED** [review]

File: docs/source/ko/main_classes/onnx.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 🤗 트랜스포머 모델을 ONNX로 내보내기[[exporting--transformers-models-to-onnx]]
```
Comment (nuatmochoi) [inline docs/source/ko/main_classes/onnx.md:17]:
```suggestion
# 🤗 Transformers 모델을 ONNX로 내보내기[[exporting--transformers-models-to-onnx]]
```

File: docs/source/ko/main_classes/onnx.md (hunk 6 above)
Comment (nuatmochoi) [inline docs/source/ko/main_classes/onnx.md:19]:
```suggestion
🤗 Transformers는 `transformers.onnx` 패키지를 제공하며, 패키지를 통해 설정 객체를 활용함으로써 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
```

File: docs/source/ko/main_classes/onnx.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 패키지를 통해 설정 객체를 활용함으로써 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
+
+🤗 트랜스포머에 대한 자세한 내용은 [이 가이드](../serialization)를 참조하세요.
```
Comment (nuatmochoi) [inline docs/source/ko/main_classes/onnx.md:21]:
```suggestion
🤗 Transformers에 대한 자세한 내용은 [이 가이드](../serialization)를 참조하세요.
```

**Review by nuatmochoi — CHANGES_REQUESTED** [review]
다른 트랜스포머 문서에서는 Transformers를 따로 번역하지 않았던 것으로 기억해서 수정해보았습니다.

File: docs/source/ko/main_classes/onnx.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+# 🤗 트랜스포머 모델을 ONNX로 내보내기[[exporting--transformers-models-to-onnx]]
+
+🤗 트랜스포머는 `transformers.onnx` 패키지를 제공하며, 이 패키지는 설정 객체를 활용하여 모델 체크포인트를 ONNX 그래프로 변환할 수 있게 합니다.
+
+🤗 트랜스포머에 대한 자세한 내용은 [이 가이드](../serialization)를 참조하세요.
+
+## ONNX 설정[[onnx-configurations]]
+
+내보내고자 하는 모델 아키텍처의 유형에 따라 상속받아야 할 세 가지 추상 클래스를 제공합니다:
```
Comment (fabxoe) [inline docs/source/ko/main_classes/onnx.md:25]:
```suggestion
내보내려는(export) 모델 아키텍처의 유형에 따라 상속받아야 할 세 가지 추상 클래스를 제공합니다:
```

File: docs/source/ko/main_classes/onnx.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+[[autodoc]] onnx.config.OnnxConfigWithPast
+
+### OnnxSeq2SeqConfigWithPast[[OnnxSeq2SeqConfigWithPast]]
+
+[[autodoc]] onnx.config.OnnxSeq2SeqConfigWithPast
+
+## ONNX 특징[[onnx-features]]
+
+각 ONNX 설정은 다양한 유형의 토폴로지나 작업에 대해 모델을 내보낼 수 있게 해주는 _features_ 세트와 연관되어 있습니다.
```
Comment (fabxoe) [inline docs/source/ko/main_classes/onnx.md:45]:
```suggestion
각 ONNX 설정은 다양한 유형의 토폴로지나 작업에 대해 모델을 내보낼 수 있게(exporting) 해주는 _features_ 세트와 연관되어 있습니다.
```

**Review by stevhliu — APPROVED** [review]
💯

**Pull request merged**